com-scrape/Cargo.toml

```diff
@@ -1,11 +1,11 @@
-[package]

```

Comment:
I just pushed a commit fixing all the line endings in the repo, so if you rebase onto latest `master` it should clean up this diff.

Reply:
Thanks! Just rebased that in